src/main/java/de/torui/coflsky/EventHandler.java

```diff
@@ -188,4 +202,165 @@ private static void ProcessScoreboard(String line){
+        if (stack != null) {
+            try {
+                String uuid = stack.serializeNBT().getCompoundTag("tag").getCompoundTag("ExtraAttributes")
+                        .getString("id");
+                if (uuid.length() == 0) {
+                    throw new Exception();
+                }
+                return uuid;
+            } catch (Exception e) {
+            }
+        }
+        return "";
+    }
+    public static String ExtractUuidFromItemStack(ItemStack stack) {
+        if (stack != null) {
+            try {
+                String uuid = stack.serializeNBT().getCompoundTag("tag").getCompoundTag("ExtraAttributes")
+                        .getString("uuid");
+                if (uuid.length() == 0) {
+                    throw new Exception();
+                }
+                return uuid;
+            } catch (Exception e) {
+            }
+        }
+        return "";
+    }
+
+    public static void GuiEventHandler(GuiOpenEvent event) {
+        if (event.gui instanceof GuiContainer/* && GuiScreen.isShiftKeyDown()*/) {
+            new Thread(() -> {
+                try {
+                    Thread.sleep(50);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+                InventoryWrapper wrapper = new InventoryWrapper();
+
+                GuiContainer gc = (GuiContainer) event.gui;
+
+                if (event.gui instanceof GuiChest) {
+                    ContainerChest chest = (ContainerChest) ((GuiChest) event.gui).inventorySlots;
+
+                    IInventory inv = chest.getLowerChestInventory();
+                    if (inv.hasCustomName()) {
+                        // verify that the chest actually has a custom name
+                        String chestName = inv.getName();
+                        wrapper.chestName = chestName;
+                    }
+                }
+
+                NBTTagCompound compound = new NBTTagCompound();
+                NBTTagList tl = new NBTTagList();
+
+                for (Slot obj : gc.inventorySlots.inventorySlots) {
+                    ItemStack stack = obj.getStack();
+                    if (stack != null) {
+                        tl.appendTag(stack.serializeNBT());
+                    } else {
+                        tl.appendTag(EMPTY_COMPOUND);
+                    }
+                }
+
+                ByteArrayOutputStream baos = new ByteArrayOutputStream();
+                try {
+                    compound.setTag("i", tl);
+                    CompressedStreamTools.writeCompressed(compound, baos);
+
+                    wrapper.fullInventoryNbt = Base64.getEncoder().encodeToString(baos.toByteArray());
+
+                    String data = WSClient.gson.toJson(wrapper);
+                    String info = QueryServerCommands.PostRequest("https://sky.coflnet.com/api/mod/description/modifications", data);
```

Comment:
this post request is missing the header values for account uuid/name and/or the coflnet session id (to find and apply settings such as which things to display)  
I can also add the parameters to the body if that would make more sense/be easier to do